addons/tmf_loadouts/loadouts/ru_army_template_2020.hpp

```diff
@@ -0,0 +1,700 @@
+	displayName = "GMG Team Leader";
+	backPack[] = {BACKPACK_SHOVEL_CLASS(CUP_O_RUS_Patrol_bag)};
+	secondaryWeapon[] = {"ace_csw_sag30CarryTripod"};
+	backpackItems[] = {
+		LIST_7("CUP_compats_29Rnd_30mm_AGS30_M"),
+		"ACE_EntrenchingTool"
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_RPG7V"};
+	secondaryAttachments[] = {"cup_optic_pgo7v3"};
+	magazines[] += {
+		"CUP_PG7VR_M",
+		"CUP_PG7VL_M"
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	backpackItems[] = {
+		"CUP_PG7VR_M",
+		"CUP_PG7VL_M"
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		"CUP_PG7VL_M",
+		LIST_2("CUP_OG7_M")
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Kornet_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Kornet_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_Kornet_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ACE_Vector"};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"CUP_2b14_carry"};
+	backPack[] = {BACKPACK_CLASS(CUP_O_RUS_Patrol_bag)};
+	backpackItems[] = {
+		LIST_3("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {BACKPACK_CLASS(CUP_O_RUS_Patrol_bag)};
+	backpackItems[] = {
+		LIST_5("ACE_1Rnd_82mm_Mo_HE")
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {BACKPACK_CLASS(CUP_O_RUS_Patrol_bag)};
+	backpackItems[] = {
+		LIST_3("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_Igla"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_Igla"};
+	linkedItems[] += {"Binocular"};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	uniform[] = {GHILLIE_CLASS};
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	headgear[] = {"CUP_H_RUS_Bandana_GSSh_Headphones"};
+	primaryWeapon[] = {"CUP_srifle_ksvk"};
+	scope[] = {"cup_optic_pso_3_open"};
+	sidearmWeapon[] = {"hgun_Rook40_F"};
+	sidearmAttachments[] = {"muzzle_snds_l"};
+	magazines[] = {
+		LIST_6("CUP_5Rnd_127x108_KSVK_M"),
+		"SmokeShellGreen",
+		"SmokeShell",
+		LIST_3("16Rnd_9x21_green_Mag")
+	};
```

Comment:
Nah I'll go with the KSVK/ASVK.

Reply:
I'd still recommend giving them a different primary though, or at least giving the spotter a proper primary instead of a sniper rifle so that way they have a fighting chance if they get caught out. The AS(M) Val is part of the new Ratnik system and is being issued to special teams like Snipers so that would be a good option.

Reply:
The spotter was intended to have a primary, just didn't spot that you had changed the inheritance in your file.